pkg/operatorstatus/operator_status.go

```diff
@@ -171,16 +172,58 @@ func (r *ClusterOperatorStatusClient) SyncStatus(ctx context.Context, co *config
 	return nil
 }
 
+func platformToInfraPrefix(platform configv1.PlatformType) string {
+	switch platform {
+	case configv1.BareMetalPlatformType:
+		return "Metal3"
+	default:
+		return string(platform)
+	}
+}
+
 func (r *ClusterOperatorStatusClient) relatedObjects() []configv1.ObjectReference {
-	// TBD: Add an actual set of object references from getResources method
-	return []configv1.ObjectReference{
+	references := []configv1.ObjectReference{
 		{Resource: "namespaces", Name: controllers.DefaultManagedNamespace},
 		{Group: configv1.GroupName, Resource: "clusteroperators", Name: controllers.ClusterOperatorName},
 		{Resource: "namespaces", Name: r.ManagedNamespace},
-		{Group: "", Resource: "serviceaccounts", Name: "cluster-capi-operator"},
-		{Group: "", Resource: "configmaps", Name: "cluster-capi-operator-images"},
-		{Group: "apps", Resource: "deployments", Name: "cluster-capi-operator"},
+		{Group: "", Resource: "serviceaccounts", Name: "cluster-capi-operator", Namespace: controllers.DefaultManagedNamespace},
+		{Group: "", Resource: "configmaps", Name: "cluster-capi-operator-images", Namespace: controllers.DefaultManagedNamespace},
+		{Group: "apps", Resource: "deployments", Name: "cluster-capi-operator", Namespace: controllers.DefaultManagedNamespace},
+		{Group: "cluster.x-k8s.io", Resource: "clusters", Namespace: controllers.DefaultManagedNamespace},
+		{Group: "cluster.x-k8s.io", Resource: "machines", Namespace: controllers.DefaultManagedNamespace},
```

Comment:
Could you please change to `r.ManagedNamespace` instead of `controllers.DefaultManagedNamespace` here and at line 186?

Reply:
Done